### PR TITLE
add openshift definitions

### PR DESCRIPTION
**Pull request by JamesKunstle**
Huge thanks to Cory Latschkowski <koree@redhat.com> for helping me gitops-ify our app.

### CODE REVIEW

**Comment by JamesKunstle** [comment]
@codekow These files define the application- where should we define the secrets? I'd assume that defining them in a private repo and applying them would be reasonable.

**Comment by codekow** [comment]
> @codekow These files define the application- where should we define the secrets? I'd assume that defining them in a private repo and applying them would be reasonable.

That's the simplest solution for now. 👍

**Comment by codekow** [comment]
@JamesKunstle To match the `docker-compose` I would refactor `web-server` to `app-server`

**Review by cdolfi — APPROVED** [review]
lgtm coming from a naive understanding of this work

File: openshift/8k-web-server.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"eightknot-app:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"eightknot-app\")].image","pause":"false"}]'
+  labels:
+    app: eightknot-web-server
+    app.kubernetes.io/component: eightknot-web-server
+    app.kubernetes.io/instance: eightknot-web-server
+    app.kubernetes.io/name: eightknot-web-server
+    app.kubernetes.io/part-of: eightknot-app
+    app.openshift.io/runtime: python
+    app.openshift.io/runtime-version: 3.9-ubi8
+  name: eightknot-web-server
```
[the remark this answers is not in the document]
Comment (JamesKunstle) [reply openshift/8k-web-server.yaml]: @codekow updated!

**Pull request merged**